### ISSUE TITLE
TwigException: Cannot extend an inline template.

### ISSUE THREAD

**Issue by javipuche** [jabney/render-template-loader#1]
```bash
Error parsing twig template undefined:
TwigException: Cannot extend an inline template.
```
I'm using example config. This error appear when use include or extends.

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Document</title>
</head>
<body>
    {% block content %}
        <div>{{username}}</div>
    {% endblock %}

    {% include 'partials/header.twig' %}
</body>
</html>
```

**Comment by jabney** [comment]
I suspect that Twig wants some extra configuration information in order to render partials. So if you remove the `{% include 'partials/header.twig' %}` and it works, then my suspicion is correct. As per the documentation for `render-template-loader`, I have only tested partials rendering with ejs, handlebars, jade, and pug.

[The tests](https://github.com/jabney/render-template-loader/tree/master/test) shows how it's done with other engines (see the tests for ejs, handlebars, jade, and pug).

There should be a way to pass configuration options to Twig but I haven't figured out how yet. I'll look into it more this weekend. If you know how Twig for `node` accepts configuration, and what options enable partials rendering, let me know.

**Comment by jabney** [comment]
This has been addressed in `v1.0.1`.  See this [test](https://github.com/jabney/render-template-loader/blob/master/test/twig.spec.js).

Here's how to make it work:

Configure the `path` option:

```javascript
const options = {
    engine: 'twig',
    locals: {
      title: 'Twig Template',
      desc: 'A template rendered by Twig'
    },
    engineOptions: function (info) {
      return { path: info.filename }
    }
  }
```

The `path` configuration will set up `twig` to look for partials relative to the source template directory. 

Specify the partial path relative to the source template:

```html
<h1>{{ title }}</h1>
{% include 'partials/desc.twig' %}
```
